label_studio/mypy.ini

```diff
@@ -0,0 +1,15 @@
+[mypy]
+plugins =
+    mypy_django_plugin.main
+check_untyped_defs = True
+explicit_package_bases = True
+strict = True
+cache_dir = ../data/.mypy_cache
```

Comment:
Hopefully this works for the folks on the team who don't develop within Docker?

Reply:
I ran it natively and it's working fine for me